modules/javafx.graphics/src/main/resources/com/sun/glass/ui/gtk/WindowDecorationGnome.css

```diff
@@ -0,0 +1,126 @@
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+.window-button-container {
+    -fx-button-placement: right;
+    -fx-allow-rtl: true;
+}
+
+.minimize-button,
+.maximize-button,
+.close-button {
+    -fx-background-color: #00000009;
+    -fx-background-radius: 15;
+    -fx-background-insets: 6;
+    -fx-pref-width: 36;
+    -fx-pref-height: 36;
+}
+
+.minimize-button.dark,
+.maximize-button.dark,
+.close-button.dark {
+    -fx-background-color: #ffffff09;
+}
+
+.minimize-button:active,
+.maximize-button:active,
+.close-button:active {
+    -fx-background-color: #00000015;
+}
+
+.minimize-button.dark:active,
+.maximize-button.dark:active,
+.close-button.dark:active {
+    -fx-background-color: #ffffff15
+}
+
+.minimize-button:active:hover,
+.maximize-button:active:hover,
+.close-button:active:hover {
+    -fx-background-color: #00000025;
+}
+
+.minimize-button.dark:active:hover,
+.maximize-button.dark:active:hover,
+.close-button.dark:active:hover {
+    -fx-background-color: #ffffff25;
+}
+
+.minimize-button:pressed,
+.maximize-button:pressed,
+.close-button:pressed {
+    -fx-background-color: #00000035 !important;
+}
+
+.minimize-button.dark:pressed,
+.maximize-button.dark:pressed,
+.close-button.dark:pressed {
+    -fx-background-color: #ffffff35 !important;
+}
+
+.minimize-button > .glyph,
+.maximize-button > .glyph,
+.close-button > .glyph {
+    -fx-background-color: #777;
+    -fx-background-insets: 6 -6 -6 6;
+    -fx-scale-shape: false;
+    -fx-position-shape: false;
+}
+
+.minimize-button:active > .glyph,
+.maximize-button:active > .glyph,
+.close-button:active > .glyph {
+    -fx-background-color: #333;
+}
+
+.minimize-button.dark:active > .glyph,
+.maximize-button.dark:active > .glyph,
+.close-button.dark:active > .glyph {
+    -fx-background-color: white;
+}
+
+.maximize-button:disabled {
+    -fx-background-color: transparent;
+}
+
+.maximize-button:disabled > .glyph {
+    -fx-background-color: #777;
+}
+
+.minimize-button > .glyph {
+    -fx-shape: "m 8,13 v 1 h 8 v -1 z";
+}
+
+.maximize-button > .glyph {
+    -fx-shape: "M 15,8.934 V 15 H 9 V 8.934 Z M 8,8 v 8 h 8 V 8 Z";
+}
+
+.maximize-button.restore > .glyph {
+    -fx-shape: "M 10 7 L 10 8 L 16 8 L 16 14 L 17 14 L 17 7 L 10 7 z M 8 9 L 8 16 L 15 16 L 15 9 L 8 9 z M 9 9.9238281 L 14 9.9238281 L 14 15 L 9 15 L 9 9.9238281 z";
+}
+
+.close-button > .glyph {
+    -fx-shape: "m 8.1464844,8.1464844 a 0.5,0.5 0 0 0 0,0.7070312 L 11.292969,12 8.1464844,15.146484 a 0.5,0.5 0 0 0 0,0.707032 0.5,0.5 0 0 0 0.7070312,0 L 12,12.707031 l 3.146484,3.146485 a 0.5,0.5 0 0 0 0.707032,0 0.5,0.5 0 0 0 0,-0.707032 L 12.707031,12 15.853516,8.8535156 a 0.5,0.5 0 0 0 0,-0.7070312 0.5,0.5 0 0 0 -0.707032,0 L 12,11.292969 8.8535156,8.1464844 a 0.5,0.5 0 0 0 -0.7070312,0 z";
```

Comment:
minor: we could probably minimize the path by rounding to 2 or 3 decimal points
(same comment for all .css changes)